### PR TITLE
Changes needed to synchronize CNML layer from Barcelona

### PR DESCRIPTION
**Pull request by PabloCastellano**

### CODE REVIEW

File: nodeshot/networking/net/models/choices.py [hunk 1]
```diff
@@ -16,7 +16,14 @@
 
 DEVICE_TYPES = {
     'radio device': 'radio',
+    'adsl': 'ADSL',
+    'cam': 'cam',
```
Comment (nemesisdesign) [inline nodeshot/networking/net/models/choices.py]: what's cam?

File: nodeshot/networking/net/models/choices.py [hunk 2]
```diff
@@ -16,7 +16,14 @@
...
+    'phone': 'phone',
+    'cloudy': 'cloudy',
```
Comment (nemesisdesign) [inline nodeshot/networking/net/models/choices.py]: what's cloudy?

File: nodeshot/networking/net/models/choices.py [hunk 3]
```diff
@@ -16,7 +16,14 @@
...
     'server': 'server',
+    'confine': 'confine',
+    'nat': 'nat',
```
Comment (nemesisdesign) [inline nodeshot/networking/net/models/choices.py]: what's nat?

**Comment by nemesisdesign** [comment]
:+1:

**Comment by PabloCastellano** [comment]
- Cam: camera streaming video.
- Cloudy: is some linux distribution http://ca.wiki.guifi.net/wiki/Qu%C3%A8_%C3%A9s_Cloudy
- Nat: network address translation

I'm not sure about what nat and cloudy represent.

**Comment by nemesisdesign** [comment]
I was thinking that in the future we could abstract these types a bit more, like "server", "linux-container", "router", "access_point", ecc.
For more specific things we could add tags, so one could tag his device as "cloudy", "raspberry" or whatever :-)

**Comment by PabloCastellano** [comment]
Definitely!

**Pull request merged**